components/courses/app/helpers/admin/courses/progress_helper.rb

```diff
@@ -59,13 +59,11 @@ def set_lecture_presence(season, lecture, student)
       end
 
       def homework_mark(lecture, student)
-        return '-' unless homework_present?(lecture, student)
```

Comment:
that was a good idea, left view things for view helpers.
and finders/services should not do this